spec/controllers/items_controller_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe ItemsController, type: :controller do
+
+  describe "GET #index" do
+    it "returns http success" do
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+
+    it "should assigns items as an empty array" do
+      get :index
+
+      expect(assigns[:items]).to be_empty
+    end
+    
+    context 'When given keyword' do
+      let(:keyword) { 'ruby' }
+      let(:items) { create_list(:rws_ichiba_item, 10) }
+
+      before do
+        expect(RakutenWebService::Ichiba::Item).to receive(:search).with(keyword: keyword, imageFlag: 1).and_return(items)
```

Comment:
__[rubocop]__
* Line is too long. [122/80]

<!-- Comment by [SideCI](https://sideci.com) -->